client/src/components/RecordAudioView/RecordAudioView.js

```diff
@@ -62,10 +61,9 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     height: 50,
     borderRadius: 10,
-    paddingHorizontal: 10,
   },
-  completeLeftContainer: {
-    backgroundColor: colors.gray.light,
+  completeRightContainer: {
+    backgroundColor: '#D6D6D6',
```

Comment:
Can you add this color to `theme/colors.js`?

Reply:
the whole color scheme in the theme is kinda messed up, (e.g. some of the "light" colors are darker than the "dark colors), i think fixing that might be a new issue

but i added it to the colors, resolved